src/kouta_indeksoija_service/indexer/job.clj

```diff
@@ -33,7 +33,7 @@
           organisaatio-changes (organisaatio/find-last-changes last-modified)
           eperuste-changes (eperuste/find-changes last-modified)
           changes-since (clojure.set/union organisaatio-changes eperuste-changes)] ;tarjonta-changes
-      (when-not (nil? changes-since)
+      (when-not (or (nil? changes-since) (= 0 (count changes-since)))
```

Comment:
Toimiiko tämä eri tavalla kuin `(when (empty? changes-since))`?

Reply:
Toimii eri tavalla. :D Mutta (when-not (empty? changes-since)) toimii myös tässä. Korjattu.